ros2cli/ros2cli/node/strategy.py

```diff
@@ -23,10 +23,12 @@
 class NodeStrategy:
 
     def __init__(self, args):
-        if is_daemon_running(args):
+        use_daemon = not getattr(args, 'no_daemon', False)
```

Comment:
Why use `getattr` here instead of `args.no_daemon`?

Reply:
A CLI implementation may not (yet) add the argument. Using `getattr` helps prevent failures -- just like it is done in [`DirectNode`](https://github.com/ros2/ros2cli/blob/b5349eefc6206e733f082d5de56d91583a0f70ce/ros2cli/ros2cli/node/direct.py#L34-L46) already.